catalog-rest-service/src/main/java/org/openmetadata/catalog/resources/dqtests/TestDefinitionResource.java

```diff
@@ -154,9 +155,17 @@ public ResultList<TestDefinition> list(
               schema = @Schema(implementation = Include.class))
           @QueryParam("include")
           @DefaultValue("non-deleted")
-          Include include)
+          Include include,
```

Comment:
@harshach trying to filter by `testPlatforms` is not working on my end. Here is how I am doing it `http://localhost:8585/api/v1/testDefinition?testPlatforms=Others`. Am i missing something?

Reply:
@TeddyCr I didn't implemented that filter yet. I'll add it today